trigger/host.go

```diff
@@ -181,13 +183,22 @@ func hostExpirationSlackPayload(t hostTemplateData, messageString string, select
 	return &notification.SlackPayload{
 		Body: msg,
 		Attachments: []message.SlackAttachment{{
-			Title:     "Spawnhost Page",
+			Title:     linkTitle,
 			TitleLink: t.URL,
 			Color:     evergreenSuccessColor,
 		}},
 	}, nil
 }
 
 func (t *hostTriggers) hostExpiration(sub *event.Subscription) (*notification.Notification, error) {
-	return t.generate(sub, expiringHostTitle, expiringHostBody)
+	timeZone := time.Local
+	if sub.OwnerType == event.OwnerTypePerson {
+		userTimeZone, err := getUserTimeZone(sub.Owner)
+		grip.Error(errors.Wrap(err, "problem getting time zone"))
```

Comment:
I would use message.Fields to make this more easily searchable

Reply:
done